### DISCUSSION TITLE
add a harvester name in config

### DISCUSSION THREAD

**Discussion by olinkcloud** [Chia-Network/chia-blockchain#9285] [Ideas]
hello,
Is it possible add a harvester name in config, and show it in farmer's summary.
It will useful for identifying the harvester in case something is not working right. especially we have lots of harvester.

Thanks.

**Comment by eric998101** [comment]
Exactly, especially when you join a pool, only can see bunch of characters as harvester name, like 4609EF42FFAC90 ... ... , it's difficult to map to physical harvester machine when you have lots of harvester.

**Comment by geotek5** [comment]
would also be nice if you have a lot of harvesters to go by a name rather than the IP when one drops off (would also be great to have some sort of notice on the farmer that a harvester disappeared)
